src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fixed `merge` command producing runs per rule to the expected runs per tool. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
```

Comment:
>er rule to the expected r [](http://example.com/codeflow?start=48&length=25)

this doesn't describe the adverse condition precisely or the state once resolved

Reply:
updated as suggested